specification.md

```diff
@@ -194,6 +208,32 @@ Represents person registration data.
 }
 ```
 
+### RegistrationInfo
```

Comment:
How can we convey if registration is being handled by an external website? I understand we likely can't convey everything but it might be good to think about this.

Reply:
I think the only thing we can do right now is a boolean? The database has an external website but that doesn't necessarily mean it's the registration website. I'm inclined to leave it out for now but let me know if you disagree.

Reply:
Just kindly pointing out that the website also has a boolean in the database called `use_wca_registration`, which I believe is exactly what Cailyn was asking for

Reply:
Ah got it, added.